app-frontend/src/assets/styles/sass/_shame.scss

```diff
@@ -3581,6 +3586,17 @@ rf-selected-actions-bar {
     }
 }
 
+.filter-pane-card {
```

Comment:
I think this ends up being unused.

Reply:
Resolved in https://github.com/raster-foundry/raster-foundry/pull/4733/commits/32aea7b0aa17f69614ec7a1feb2cb5480d58f712